data-table/table.js

```diff
@@ -305,6 +306,47 @@ function createRowEntryBox(config) {
     createRowDeleteBtn(config);
 }
 
+/**
+ * This function enables the user to enter the name of a column. Creates a field for text input
+ * as well as a button that sends input text to addSingleColumn().
+ * @param {object} config   - Table configuration object
+ * @returns {undefined}     - Doesn't return anything
+ */
+function createColumnInputAndBtn(config) {
+    let entryBoxDiv = document.getElementById(config.tableIds.entryBoxDivId);
+    // Creates the field for text input
+    let input = document.createElement("INPUT");
+    input.type = 'text';
+    input.placeholder = "Enter " + config.datumConfig.names;
```

Comment:
`datumConfig` is for the specific value type - think % chance of precipitation, or numVotes.

You want `config.columnsName` here, I think?

Reply:
Yup, totally right. Made the change.